docs/sources/datasources/graphite/template-variables/index.md

```diff
@@ -37,122 +37,157 @@ refs:
 # Graphite template variables
 
 Instead of hard-coding details such as server, application, and sensor names in metric queries, you can use variables.
-Grafana lists these variables in dropdown select boxes at the top of the dashboard to help you change the data displayed in your dashboard.
+Grafana lists these variables in drop-down selection boxes at the top of the dashboard to help you change the data displayed in your dashboard.
 Grafana refers to such variables as template variables.
 
 For an introduction to templating and template variables, refer to the [Templating](ref:variables) and [Add and manage variables](ref:add-template-variables) documentation.
 
-## Select a query type
+To view an example templated dashboard, refer to [Graphite Templated Nested dashboard](https://play.grafana.org/d/cvDFGseGz/graphite-templated-nested).
+
+## Use query variables
+
+With Graphite, the most commonly supported variables are query variables.
+
+Grafana supports three query types specifically for Graphite-based template variables:
+
+| Query type            | Description                                                                                                                                                     | Example usage                            |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------- |
+| **Default query**     | Allows you to dynamically list metrics, nodes, or tag values using Graphite functions such as `tags()`, `tag_values(<metric>, <tag>)`, `expand(<metric>)`, etc. | `tag_values(apps.*.requests.count, app)` |
+| **Value query**       | Returns the unique values for a specific tag key within a given metric series, often used to populate drop-downs with tag values.                               | `tag_values(apps.*.status.*, status)`    |
```

Comment:
Are the drop-downs referring to selecting a value for the template variable? If so, this is the _only_ way to use this. I think I'm not understanding something :smiley:

Reply:
Do we need to specify "metric and function"? My knowledge of Graphite isn't the best.
I was thinking it could just say "Returns all the unique values from a query."